main/main.ino

```diff
@@ -1791,6 +1794,19 @@ void stateMeasures() {
 }
 #endif
 
+#if defined(ESP8266) || defined(ESP32)
+void SyncNTP() {
```

Comment:
Could you use this function below to avoid the block duplication Row 2110

Reply:
I used the SyncNTP in place of the (mostly duplicated) block @ row 2110, however I was not 100% sure of the startProcessing() call, that's why it was not originally in SyncNTP().
Also, I've tried in on my ESP32 and it's working well, (so far, with more than 6 days of uptime) but I think it could be overkill to NTP sync every 2 minutes.. (as I'm doing now) maybe the Sync could be called.. say, every hour instead? (just an idea, did not check yet how reliable ESP32 timing is)